src/index.js

```diff
@@ -1,11 +1,12 @@
 import MascotBot from './bots/mascot-bot.js';
-import GreetOnStart from './behaviors/greet-on-start.js';
+import TestBehavior from './behaviors/test-behavior.js';
```

Comment:
TestBehavior doesn't exist. Why did you take out GreetOnStart? I thought it was a good example.